src/HealthGPS.Input/model_parser.cpp

```diff
@@ -24,29 +24,30 @@
 
 namespace hgps::input {
 
-hgps::RiskFactorSexAgeTable load_risk_factor_expected(const Configuration &config) {
+std::unique_ptr<hgps::RiskFactorSexAgeTable>
+load_risk_factor_expected(const Configuration &config) {
```

Comment:
I suppose this *could* be a member function...

Reply:
Of the `AdjustableRiskFactorModelDefinition` class, you mean?